openreview/openreview.py

```diff
@@ -20,17 +21,17 @@ class OpenReviewException(Exception):
     pass
 
 class Client(object):
-
+    """
+    :param baseurl: URL to the host, example: https://openreview.net (should be replaced by 'host' name)
+    :type baseurl: str, optional
+    :param username: OpenReview username
+    :type username: str, optional
+    :param password: OpenReview password
+    :type password: str, optional
+    :param token: Session token
+    :type token: str, optional
```

Comment:
What is a token in this case?  What does it do?

Reply:
I believe this is the Authentication token. I can be more specific and add this